modules/migration-mapping-destination-namespaces-in-the-migplan-cr.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * migrating_from_ocp_3_to_4/advanced-migration-options-3-4.adoc
+
+[id="migration-mapping-destination-namespaces-in-the-migplan-cr_{context}"]
+= Mapping destination namespaces in the MigPlan custom resource
+
+If you map destination namespaces in the `MigPlan` cust resource (CR), you must ensure that the namespaces are not duplicated on the source or the destination clusters because the UID and GID ranges of the namespaces are copied during migration.
```

Comment:
```suggestion
If you map destination namespaces in the `MigPlan` custom resource (CR), you must ensure that the namespaces are not duplicated on the source or the destination clusters because the UID and GID ranges of the namespaces are copied during migration.
```

Reply:
Thanks for catching that. I'll add it in my own commit so that I don't have to squash later.